File: examples/src/Text/HighlightText.cs
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using Aspose.slides.Export
using System
using System.Collections.Generic
import aspose.pydrawing as drawing
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.text
{
    class HighlightText
    {
        public static void Run()
        {

            #ExStart:HighlightText
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Text()
            Presentation presentation = new Presentation(dataDir +"SomePresentation.pptx")
            ((AutoShape)presentation.slides[0].shapes[0]).text_frame.HighlightText("title", Color.LightBlue) # highlighting all words 'important'
            ((AutoShape)presentation.slides[0].shapes[0]).text_frame.HighlightText("to", Color.Violet, new TextHighlightingOptions()
            {
                WholeWordsOnly = True
            }) # highlighting all separate 'the' occurrences
            presentation.save(dataDir+ "SomePresentation-out2.pptx", slides.export.SaveFormat.PPTX)

            #ExEnd:HighlightText
        }
    }
}

File: examples/src/text/highlight_text.py
import aspose.slides as slides
import aspose.pydrawing as drawing


def highlight_text(global_opts):
    with slides.Presentation(global_opts.data_dir + "text_default_fonts.pptx") as presentation:
        options = slides.TextSearchOptions()
        options.whole_words_only = True
    
        presentation.slides[0].shapes[0].text_frame.highlight_text("title", drawing.Color.light_blue)
        presentation.slides[0].shapes[0].text_frame.highlight_text("to", drawing.Color.violet, options, None)

        presentation.save(global_opts.out_dir + "text_highlight_text_out.pptx", slides.export.SaveFormat.PPTX)
